zerothi/sisl
# Sisl visualization and graphical interface

What is this
-----
This pull request is to get visibility, so that people knows that this is under developement and I can (hopefully) get some feedback about it. Many syntax things still need to be changed, but the concept is already quite complete, so I'd like to get feedback on it to improve weak points.

For those who don't know, this project is an attempt to build a plotting framework and a graphical interface that can really display the great potential of sisl for processing scientific data.

How can I test it
----
It should be straightforward for sisl users to test it, I put everything needed in my branch (although I know that in production I will have to store it elsewhere) so that it is as simple as:

```
git clone -b GUI https://github.com/pfebrer96/sisl.git
cd sisl
pip install -e .[visualization]
```

for anyone that would like to test it.

Where do I start
------
Under `sisl/viz/Notebooks/tutorials` there are 3 tutorial notebooks that I hope can be quite self-explanatory on introducing people to the framework. Actually I would like feedback on them too :).

I would appreciate it if (and I recommend) you to go trough the `Demo` and the `GUI with Python Demo` in that order. I think that would be enough to get you an idea of the concept.

The notebooks are also here: https://github.com/pfebrer96/sisl.viz-tutorials. But for testing convenience I will keep them inside the package too for now.

**Thanks in advance to any person that takes the time to check it out!**

Cheers!
